### DISCUSSION TITLE
How to configure openrouter and claude

### DISCUSSION THREAD

**Discussion by yxd0018** [unit-mesh/auto-dev#214] [Q&A] [answered]
Can you anyone give example how to config for openrouter and claude with API key? The doc should have that as example for custom AI engine.

**Comment by phodal** [comment, marked as answer]
Refs https://github.com/unit-mesh/auto-dev/issues/209

**Reply by yxd0018** [reply]
what about openrouter?

**Reply by phodal** [reply]
> what about openrouter?

https://ide.unitmesh.cc/custom/llm-server
